src/components/Admin/ListSkills/ListSkills.js

```diff
@@ -571,12 +572,13 @@ class ListSkills extends React.Component {
       {
         title: 'Name',
         dataIndex: 'skillName',
-        width: '20%',
+        width: window.innerWidth <= 520 ? '30vw' : '20%',
```

Comment:
This logic for checking innerWidth is repetitive. Can you add it in a util function?

Reply:
`520` should be factorized out, in case one wants to change it at some point in the future.